examples/window-app/silabs/src/WindowManager.cpp

```diff
@@ -742,6 +742,15 @@ void WindowManager::ButtonEventHandler(uint8_t button, uint8_t btnAction)
     }
 
     AppTask::GetAppTask().PostEvent(&event);
+
+// Since there is no release interrupt for button1 posting release event after press event
+#ifdef SIWX_917
```

Comment:
This should be moved to the spam Layer (The SPAM layer for the 917 should have a workaround in the button handler or something of the sort)
OR 
Just add a Note in the README stating that problem with the 917. Since it's a API feature mismatch between two chips, we shouldn't have patch like this one in the Application layer. A ticket should be raise to the platform team to fix this.

Reply:
Also this fix seems to mess around with the behavior of the App. I would just completely removed this part and add a note inside the README stating this bug for the 917.

Reply:
This issue occurred due to the release interrupt is not handled properly for 917SoC. 
as per your suggestion will handle it from the platform layer.